source/_posts/2025-07-25-nfc-floppy-disk-game-launcher.markdown

```diff
@@ -0,0 +1,249 @@
+- **Tag collision** detection (multiple tags present)
+- **Serial timeout** management
+- **Unknown tag** logging
+- **Graceful shutdown** on interruption
+
+## Future Enhancements
+
+There are several directions this project could evolve:
+
+### Hardware Improvements
+- **LED indicators** for better user feedback
+- **Sound effects** to enhance the retro experience  
+- **Multiple readers** for a full retro computer setup
+- **Custom PCB** for more professional integration
+
+### Software Features
+- **GUI configuration** for easier game mapping
+- **Automatic game detection** from file associations
+- **Usage statistics** and favorite games tracking
+- **Network integration** for multiplayer game coordination
+
+### Gaming Integration
+- **Save state management** tied to specific disks
+- **Achievement systems** tracked per physical disk
+- **Multiplayer coordination** (different disks for different players)
+- **Game collections** (one disk launches a curated game menu)
+
+## Why This Matters
+
+This project represents something important in our increasingly digital world: the value of physical interaction with our digital content. While we've gained incredible convenience with digital libraries, we've lost the ritual and intentionality of choosing what to play.
+
+With physical game disks, you're forced to make a deliberate choice. You handle the disk, read the label, and commit to that experience. It slows down the decision-making process in a way that actually enhances the gaming experience.
+
+## Building Your Own
+
+The complete source code is available on [GitHub](https://github.com/rephus/nfc-floppy-disk-reader). The project is designed to be easily reproducible with common components.
+
+**Total cost:** Under $30 in components
+**Build time:** 4-5 hours  
+**Skill level:** Beginner-friendly (basic Arduino and Python knowledge helpful)
+
+The most time-consuming part is actually creating the custom disk labels , and building the mechanical parts in Lego - but that's also the most rewarding part!
+
+## Conclusion
+
+The NFC Floppy Disk Game Launcher proves that sometimes the best new ideas come from combining old and new technologies in unexpected ways. It creates a bridge between the tactile satisfaction of retro gaming and the convenience of modern digital libraries.
+
+More importantly, it demonstrates that innovation doesn't always mean adding complexity - sometimes it means thoughtfully removing it. By stripping away the mechanical complexity of floppy drives while preserving their physical interaction model, we get the best of both worlds.
+
+Whether you're a retro gaming enthusiast, an Arduino hobbyist, or just someone who misses the physicality of old computing, this project offers a fun way to bring some of that tactile magic back to modern gaming.
+
+*Have you built something similar? I'd love to see your take on physical digital media! Share your projects in the comments below.*
```

Comment:
El call to action creo que sobra, que yo sepa no tenemos ni comentarios en el blog.